config.tpl.yml

```diff
@@ -30,9 +30,6 @@ APPS:
     min_instances: {{ MIN_INSTANCE_COUNT_API }}
     max_instances: {{ MAX_INSTANCE_COUNT_API }}
     scalers:
-      - type: ElbScaler
```

Comment:
I like the idea of removing autoscaling syntax for API app since we do not scale it. It feels consistent.